src/Elan.Web/ClientApp/src/containers/Wall/Wall.jsx

```diff
@@ -21,21 +35,31 @@ export default class Wall extends Component {
     //do some crazy register things 
   }
 
+  post = () => {
+    savePost(this.state.postContent)
+      .then(() => {
+        this.setState({postContent: ""})
+        this.getPosts();
+        });
+  };
+
   render() {
-    //test
+    /*
```

Comment:
You can remove these lines if they are no longer needed.

Reply:
I would leave these for a while. It can be helpful for testing and useful for person responsible for return posts in api because for now api returns not enough info. I need model something like this in test data.